Userland/Applications/Presenter/SlideObject.cpp

```diff
@@ -142,10 +147,12 @@ ErrorOr<void> HTMLElement::serialize(StringBuilder& builder) const
         // FIXME: Escape the value string as necessary.
         TRY(builder.try_appendff(" {}='{}'", key, value));
     }
-    TRY(builder.try_append(" style=\""sv));
-    for (auto const& [key, value] : style) {
-        // FIXME: Escape the value string as necessary.
-        TRY(builder.try_appendff(" {}: {};", key, value));
+    if (!style.is_empty()) {
+        TRY(builder.try_append(" style=\""sv));
+        for (auto const& [key, value] : style) {
+            // FIXME: Escape the value string as necessary.
```

Comment:
escaping is misleading here, the keys and values should rather be sanitised here (and above).

Reply:
I just copied this fixme over from what was there before